### DISCUSSION TITLE
如果将类属性设置为：[AutoGenerateColumn(Ignore = true)]，第一个TableColumn会消失

### DISCUSSION THREAD

**Discussion by HongFei-Ye** [dotnetcore/BootstrapBlazor#3868] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![11](https://github.com/user-attachments/assets/1f37f6c2-4193-4304-9273-ff8dbde285e0)

如果将ProductModels设置为：[AutoGenerateColumn(Ignore = true)]，第一个TableColumn会消失

在8.7.0版本正常，能够符合预期全部显示，8.7.1版本和8.7.2版本都会出现这个问题。

详细请查看复现工程（BlazorApp2.zip）

### Expected Behavior

即使我将ProductModels设置为：[AutoGenerateColumn(Ignore = true)]，但是我有使用了TableColumn，且TableColumn的Visible="true"，TableColumn的优先级别应该高于AutoGenerateColumn，所以有多少个TableColumn就应该显示多少个。

### Steps To Reproduce

[BlazorApp2.zip](https://github.com/user-attachments/files/16260364/BlazorApp2.zip)

当前复现工程版本为8.7.0，升级到8.7.1或者8.7.2即可复现。

### .NET Version

8.0

**Comment by bb-auto** [comment]
@HongFei-Ye Thank you for reporting. We will give triage later.

**Reply by ArgoZhang** [reply, marked as answer]
发布新版本 `8.7.3-beta05` 增加 `TableTemplateColumn` 可不绑定模型任何字段

```
<TableTemplateColumn>
    <Template Context="v">
        @v.Row.Name
    </Template>
</TableTemplateColumn>
```

模板 `Template` 内可以做很多事

link #3879

**Reply by densen2014** [reply]
正是我需要的
